src/presenter/films-presenter.js

```diff
@@ -2,24 +2,22 @@ import {render, remove} from '../framework/render.js';
 import FilmsView from '../view/films-view.js';
 import FilmsListView from '../view/films-list-view.js';
 import FilmsListContainerView from '../view/films-list-container-view.js';
-import FilmCardView from '../view/film-card-view.js';
-import InfoPopUpView from '../view/info-pop-up-view.js';
 import ShowMoreButtonView from '../view/show-more-button-view.js';
 import NoFilmView from '../view/no-film-view.js';
 import TopRatedView from '../view/top-rated-view.js';
 import MostCommentedView from '../view/most-commented-view.js';
-
+import CardFilmPresenter from './card-film-presenter.js';
+import { updateFilmCard, topRatedFilmSort, commentedFilmSort } from '../utils/common.js';
 
 const FILM_CARD_PER_STEP = 5;
 const TOP_AND_MOST_COUNT = 2;
 
-const bodyElement = document.querySelector('body');
-
 
 export default class FilmsPresenter {
   #filmContainer = null;
   #filmsModel = null;
-  #filmsModelContainer = null;
+  #filmsModelContainer = [];
+  #cardFilmPresenter = new Map();
```

Comment:
```suggestion
  #cardFilmPresenters = new Map();
```
Это же всё равно коллекция, а не один презентер.